docs/content/how-tos/rules/FL0084.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FL0084
+category: how-to
+hide_menu: true
+---
+
+# DisallowShadowing (FL0084)
+
+*Introduced in `0.23.8`*
+
+## Cause
+
+A variable or parameter shadows another one with the same name.
+
+## Rationale
+
+Sometimes shadowing can cause confusion.
+
+## How To Fix
+
+Rename varaible or parameter in question so it has unique name in its scope.
+
+## Rule Settings
+
+    {
+        "disallowShadowing": {
+            "enabled": true
```

Comment:
@webwarrior-ws please make docs be in sync with our current defaults

Reply:
changed "enabled" to false